``make docs`` broken

jensens@minime:~/workspace/plone.api$ make docs
    bin/buildout 
    Develop: '/home/jensens/workspace/plone.api/.'
    [...]
    Install Git pre-commit hook.
    bin/sphinx-build -W docs docs/html
    Running Sphinx v1.1.3
    loading translations [en]... locale not available
    loading pickled environment... done
    No builder selected, using default: html
    building [html]: targets for 19 source files that are out of date
    updating environment: [config changed] 19 added, 0 changed, 0 removed
    reading sources... [100%] user                                                                                                                    
    looking for now-outdated files... none found
    pickling environment... done
    checking consistency... 
    Warning, treated as error:
    /home/jensens/workspace/plone.api/docs/CHANGES.rst:: WARNING: document isn't included in any toctree

    make: *** [docs/html/index.html] Error 1

--- a/docs/CHANGES.rst
+++ b/docs/CHANGES.rst
@@ -4,6 +4,9 @@ Changelog
 1.3.3 (unreleased)
 ------------------
 
+- Fixed `make docs`.
+  [jaroel]
+
 - explicit dependencies in setup.py, explicit zcml loading in tests.
   [jensens]
 

--- a/docs/contribute/index.rst
+++ b/docs/contribute/index.rst
@@ -39,6 +39,3 @@ Description of our release process and guidelines.
    :maxdepth: 2
 
    release.rst
-
-
-.. include:: ../CHANGES.rst

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -65,6 +65,7 @@ Contribute
     :maxdepth: 2
 
     contribute/index.rst
+    CHANGES.rst
 
 
 Indices and tables

